reframe/core/schedulers/slurm.py

```diff
@@ -562,9 +562,6 @@ def poll(self, *jobs):
                 job_match = jobinfo[job.jobid]
             except KeyError:
                 job._state = 'CANCELLED' if job.is_cancelling else 'COMPLETED'
-                if job.exitcode is None:
```

Comment:
@vkarak I removed the `exitcode` from the squeue scheduler since we cannot get it.